src/game/ingame_menu.c

```diff
@@ -1523,6 +1523,44 @@ void reset_red_coins_collected(void) {
     gRedCoinsCollected = 0;
 }
 
+LangArray textReonucam1 = DEFINE_LANGUAGE_ARRAY(
+    "CAMERA SPEED: %d",
+    "VITESSE CAMÉRA: %d",
+    "KAMERA GESCHWINDIGKEIT: %d",
+    "カメラそくど: %d",
+    "VELOCIDAD DE CÁMARA: %d");
+
+LangArray textReonucam2 = DEFINE_LANGUAGE_ARRAY(
+    "DPAD TO CHANGE",
+    "CHANGER AVEC DPAD",
+    "DPAD ZU ÄNDERN",
+    "DPADでへんこう",
+    "DPAD PARA CAMBIAR");
+
+#ifdef REONUCAM
+void render_reonucam_speed_setting(void) {
+    char buf[25];
+    gSPDisplayList(gDisplayListHead++, dl_ia_text_begin);
+    gDPSetEnvColor(gDisplayListHead++, 255, 255, 255, gDialogTextAlpha);
+    sprintf(buf, LANG_ARRAY(textReonucam1), gReonucamState.speed +1 );
+
+    print_generic_string_aligned(310, 24, buf, TEXT_ALIGN_RIGHT);
+    print_generic_string_aligned(310, 8, LANG_ARRAY(textReonucam2), TEXT_ALIGN_RIGHT);
+
+
+    gSPDisplayList(gDisplayListHead++, dl_ia_text_end);
+
+    if (gPlayer1Controller->buttonPressed & R_JPAD) {
+        gReonucamState.speed++;
+    } else if (gPlayer1Controller->buttonPressed & L_JPAD) {
+        gReonucamState.speed--;
+    }
+    gReonucamState.speed = (gReonucamState.speed + 5) % 5;
```

Comment:
Can you extern `cameraSpeeds`, give it a hard size (5), and use `ARRAY_COUNT` here instead? Would make configuring camera speeds safer. Also rename it to `gReonucamSpeeds` or something less generic.